server/tools/build-keycloak.sh

```diff
@@ -87,14 +87,29 @@ rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
 bin/jboss-cli.sh --file=/opt/jboss/tools/cli/standalone-ha-configuration.cli
 rm -rf /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
 
+###########
+# Garbage #
+###########
+
+rm -rf /opt/jboss/keycloak/standalone/tmp/auth
+rm -rf /opt/jboss/keycloak/domain/tmp/auth
+
 ###################
 # Set permissions #
 ###################
 
 echo "jboss:x:1000:jboss" >> /etc/group
 echo "jboss:x:1000:1000:JBoss user:/opt/jboss:/sbin/nologin" >> /etc/passwd
 chown -R jboss:jboss /opt/jboss
-chmod -R g+rw /opt/jboss
-
-rm -rf /opt/jboss/keycloak/standalone/tmp/auth
-rm -rf /opt/jboss/keycloak/domain/tmp/auth
+chmod -R 744 /opt/jboss
+# Enable listings
+chmod -R +X /opt/jboss
```

Comment:
@slaskawi Is there a reason, the `X` modifier in the command above needs to be in capital letter? Confirmed it works too, but wondering about the motivation, why it was chosen?

Reply:
The `X` parameter applies executable permission only to directories. This is exactly what we want to achieve here.